.scalafix.conf

```diff
@@ -0,0 +1,31 @@
+rules = [
+  DisableSyntax, # Disables some constructs that make no semantic sense like `final val`
+  ProcedureSyntax, # Procedure syntax in Scala is always discouraged
+  ExplicitResultTypes, # To avoid public API breakages by mistake is good to always annotate the return types of public methods
```

Comment:
I'm not sure we need this, as we don't promise API stability anyway

Reply:
Even though this is true, I still personally like having explicit return types (especially so if we have a tool that automatically adds it). If nothing else it improves compile speed a bit!

If you feel strongly about it I can remove it.